src/org/zalando/stups/friboo/system/audit_log.clj

```diff
@@ -122,6 +122,7 @@
   Lifecycle
 
   (start [component]
+    (log/warn "DEPRECATED use of audit-log component. Please one of the explicit audit-loggers instead.")
```

Comment:
I want to drop this once every `friboo` service here uses the `http` audit logger.